typescript/sdk/src/deploy/core/AbacusCoreDeployer.ts

```diff
@@ -156,7 +156,11 @@ export class AbacusCoreDeployer<Chain extends ChainName> extends AbacusDeployer<
       }
     });
 
-    const remotes = this.multiProvider.remoteChains(chain);
+    const configChains = Object.keys(this.configMap) as Chain[];
+    const remotes = this.multiProvider
+      .remoteChains(chain)
+      .filter((_) => configChains.includes(_));
+
```

Comment:
in hindsight I think config being larger than multiprovider should throw

Reply:
Yeah, that was originally also my thought